src/main/java/school/lemon/changerequest/java/introduction/hw2/Task3.java

```diff
@@ -4,13 +4,15 @@
 public class Task3 {
     public static String comboString(String s1, String s2) {
 
-        if (s1.length()<s2.length())
-            return s1+s2+s1;
+        if (s1 == null && s2 == null)
+            return null;
+        if (s1.length() < s2.length())
+            return s1 + s2 + s1;
 
-        else return s2+s1+s2 ;
+        else return s2 + s1 + s2;
```

Comment:
Your implementation will throw NullPointerException if either `s1` **OR** `s2` will be null, cause you'll try to call `.length()` method on `null` value.
add additional check for cases, when only one argument is `null`.